src/pois.rs

```diff
@@ -97,6 +99,7 @@ impl IndexedPoi {
         let poi = Poi {
             id,
             coord: poi_coord,
+            approx_coord: Some(approx_coord),
```

Comment:
it should be possible to write 
```suggestion
            approx_coord: Some(poi_coord.into()),
```
and remove the geojson dependency completely.

Reply:
Nice catch! I didn't  that mimir implemented required trait. I just fixed this.